lib/peek/rblineprof/syntax_highlighter.rb

```diff
@@ -0,0 +1,38 @@
+require 'peek/rblineprof/highlighters/pygments_highlighter'
+require 'peek/rblineprof/highlighters/rouge_highlighter'
+
+module Peek
+  module Rblineprof
+    class SyntaxHighlighter
+      class_attribute :highlighter
+
+      def self.highlight(code, lexer)
```

Comment:
Here we'd receive `file_name, code` and pass them to `highlighter.process`.

We could make both highlighters inherit from an abstract class in which we could put the `lexer_for_filename(file_name)` method to stay DRY?